projects/driver_controls/test/test_driver_fsm.c

```diff
@@ -28,6 +28,14 @@ static FSMGroup s_fsm_group;
 static bool powered = false;
 static bool mech_brake = false;
 
+// Print the FSM transitions. Invalid actions that don't cause transitions are not printed
+static void prv_output(FSM *fsm, EventArbiterOutputData data) {
```

Comment:
If this is the only reason you added the FSM as an argument, I don't think it's worth it.